docs/products/clickhouse/getting-started.rst

```diff
@@ -34,19 +34,17 @@ Connect to ClickHouse
 
 2. Go to the *Overview* tab and copy the **Host**, **Port**, **User**, and **Password** parameters that you need for connecting to the service.
 
-
-3. Run the following command to connect to your service and run an SQL query on your database:
+3. Run the following command to connect to your service and run SQL queries on your database:
 
    .. code:: bash
 
-       docker run --interactive \
+       docker run -it \
        --rm clickhouse/clickhouse-client \
        --user USER-NAME \
        --password USER-PASSWORD \
        --host YOUR-HOST-NAME.aivencloud.com \
```

Comment:
can we just name parameters `HOST` rather than `YOUR-HOST` I believe it's more consistent with other examples?

Reply:
Thank you for correcting these!